import-scripts/update-cdm-deliverable.sh

```diff
@@ -58,3 +63,21 @@ rm $TMP_MERGED_SEQ_DATE
 rm $CDM_DELIVERABLE
 
 echo "`date`: CDM deliverable generation and upload complete"
+
+TMP_LOG_FILE="${PORTAL_HOME}/tmp/trigger_s3_dag.log"
```

Comment:
maybe just for organization, move variables to top of the file?

Reply:
and just a quick question -- why do we need the a tmp log file? Will it not get written to stdout and into the overnight pipelines log? 

Also, do we need to remove the tmp log file?

Reply:
I wanted to capture the HTTP status code and the contents of the HTTP response, and to get both I need to store the response contents somewhere (in a file). I know that this script writes to the fetch dmp log file, but I couldn't find that path stored in any var for me to use (I feel like the crontab controls the log path?) so I created the temp file. I can add a line to remove the file afterwards